README.md

```diff
@@ -52,47 +52,62 @@ npm install @opentelemetry/api
 The Dynatrace exporter is added and set-up like this:
 
 ```js
-const { configureDynatraceMetricExport } = require('@dynatrace/opentelemetry-exporter-metrics');
+'use strict';
+
 const { diag, DiagConsoleLogger, DiagLogLevel } = require('@opentelemetry/api');
 const { Resource } = require('@opentelemetry/resources');
 const { MeterProvider } = require('@opentelemetry/sdk-metrics');
+const { configureDynatraceMetricExport } = require('@dynatrace/opentelemetry-exporter-metrics');
+
 
-// optional: set up logging for OpenTelemetry
-diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL);
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL)
```

Comment:
At least a comment that this is optional and for debugging internal things is probably a good idea so people don't blindly copy this

Reply:
Added a comment in [0cee3c9](https://github.com/dynatrace-oss/opentelemetry-metric-js/pull/37/commits/0cee3c9b92bf3df3f495dc3299059305ada0842b) :slightly_smiling_face: